style.css

```diff
@@ -1,123 +1,377 @@
+  display: flex;
+  gap: 1em;
+  justify-content: flex-start;
+  list-style-type: none;
+}
+
+.work-section {
+  background: #e5e5e5;
+  width: 100%;
+}
+
+.work-section .card {
+  width: 90%;
+  margin: 0 auto;
+  background-color: #fff;
+  border-radius: 1rem;
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto auto;
+  border: 1px solid #dfe1e6;
+}
+
+.section-img {
+  width: 90%;
+  margin: 0 auto;
+  padding: 1rem;
+  height: auto;
+}
+
+.section-img img {
+  width: 100%;
+}
+
+.card-info {
+  width: 90%;
+  margin: 0 auto;
+  height: 50vh;
 }
 
-.header {
-  width: 327px;
-  height: 150px;
+.work-header,
+.about-header {
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 700;
-  font-size: 50px;
-  line-height: 60px;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  padding-bottom: 1rem;
+}
+
+.descr-list {
+  list-style-type: none;
+  display: grid;
+  grid-template-columns: auto auto auto;
+  justify-content: flex-start;
+  column-gap: 2em;
+  padding-bottom: 1rem;
+}
+
+.descr-list-item1 {
+  width: 55px;
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.descr-list-item {
+  width: 88px;
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.counter-img {
+  padding-right: 15px;
+}
+
+.lang-descr {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  padding-bottom: 1rem;
+}
+
+.lang-list {
+  display: flex;
+  justify-content: flex-start;
+  gap: 1rem;
+  list-style-type: none;
+  width: min-content;
+  padding: 0.5rem;
+  height: 5vh;
+}
+
+.lang-list-descr {
   display: flex;
+  flex-direction: row;
   align-items: center;
-  color: #172b4d;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  padding-top: 0;
-  padding-left: 12px;
+  padding: 4px 12px;
+  height: 24px;
+  background: #ebebff;
+  border: 1px solid whitesmoke;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.btn-project,
+.btn-resume {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  left: 0;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  color: #396df2;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 550;
+  font-size: 17px;
+  line-height: 24px;
+  margin-top: 2rem;
+}
+
+.btn-project:hover,
+.btn-resume:hover {
+  background-color: royalblue;
+  color: #fff;
 }
 
-.about {
-  width: 327px;
-  height: 146px;
+.about-myself {
+  padding-top: 100px;
+  padding-bottom: 100px;
+}
 
```

Comment:
- [ ] Please from the project requirement, you need to use flex to display this about section for easy alignment of element and also make it responsive :pray: